Tasks/XamariniOS/Tests/L0.ts

```diff
@@ -14,63 +14,63 @@ describe('XamariniOS L0 Suite', function () {
 
     });
 
-    it('XamariniOS signing with identifiers', (done:MochaDone) => {
+    it('XamariniOS signing with identifiers', function (done: MochaDone) {
```

Comment:
Why change lambda expression to functions?  `=>` is shorter. :p

Reply:
https://mochajs.org/#arrow-functions
Arrow functions are not supposed to work with Mocha.  The only reason it works for us right now is because the TypeScript compiler is transforming the arrow functions to classic functions behind the scenes.  I don't think we should rely on this behavior and it will be a heck of a bug if the TypeScript folks ever change it 😉

Reply:
We have these everywhere, seems low risk of typescript breaking it

Reply:
Agreed, but why not make it no risk?  It's also confusing to have something contradicting what is stated in the Mocha documentation.

It took me about 30 min trying to figure out whether the tests were behaving properly and whether all of the Mocha functionality was available to us until I realized what was going on.

Reply:
I don't understand the point about not relying on typescript transpiling.  That's the whole point of typescript - write modern javascript (ES7+) today and let it transpile.   Node (what mocha uses) is actually moving very fast and node 8+ supports this.  I don't want to have to go back and change all our code to modern javascript later.  Use it now.  

It's also inconsistent with the rest of the code base which is moving intentionally to modern typescript / javascript.

Reply:
My point is that I don't believe that this is documented behavior by the compiler.  Even in modern JavaScript, arrow functions and classic functions / closures have different semantics and different use cases, and this is one of them.  It's not like let/const vs var where var is essentially deprecated.  But since the rest of the codebase does use this I can change it back.

Reply:
Actually, I was mistaken.  TypeScript actually doesn't turn this to classic functions, and so these are broken.  None of the calls to `this.timeout` do anything.